src/zoid/buttons/util.js

```diff
@@ -34,3 +37,17 @@ export function isSupportedNativeBrowser() : boolean {
 
     return false;
 }
+
+export function createVenmoExperiment() : Experiment {
+    return createExperiment('enable_venmo', 0);
```

Comment:
@bluepnume do these params look okay? 

1. name - My assumption is the name parameter value ("enable_venmo") is not tied to any elmo experiment. Instead, this name is used for the FTPI logs and we will need a custom dashboard to report on this experiment.
2. sample - I think a zero value for sample disables the experiment. And when we want to turn this on we will need to make a code change here.

Please let me know if I've overlooked anything.

Reply:
Totally works!